pyiceberg/io/__init__.py

```diff
@@ -82,6 +82,10 @@
 ADLS_CLIENT_ID = "adls.client-id"
 ADLS_CLIENT_SECRET = "adls.client-secret"
 ADLS_ACCOUNT_HOST = "adls.account-host"
+ADLS_BLOB_STORAGE_AUTHORITY = "adls.blob-storage-authority"
```

Comment:
doc reference:
https://arrow.apache.org/docs/python/filesystems.html#azure-storage-file-system
and
https://arrow.apache.org/docs/cpp/api/filesystem.html#azure-filesystem
👍